### PR TITLE
Enhancement: Use PSR-12 instead of PSR-2 coding standard

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] uses the `PSR12` coding standard instead of `PSR2`
* [x] runs `composer fix`

### CODE REVIEW

File: inc/nginx-helper.php [hunk 1]
```diff
@@ -1,8 +1,11 @@
 <?php
+
+declare(strict_types=1);
```
Comment (localheinz) [inline inc/nginx-helper.php]: The `Doctrine` coding standard is probably a bit to strict for this project.

With a lack of tests, I would probably not apply this standard.

**Comment by localheinz** [comment]
After running 

```
$ composer fix
```

running

```
$ composer sniff
```

fails with a ton of errors that would need to get fixed manually.

As an alternative to using `squizlabs/php_codesniffer`, we could use `friendsofphp/php-cs-fixer`.

It has the advantage that it comes with a single command line tool, `phpcs`, and only consists of fixers - that is, it will only complain about code style issues that it is capable of fixing.

**Comment by svpernova09** [comment]
I would expect us to current support PSR-2, and be moving towards PSR-12 https://www.php-fig.org/psr/psr-12/

**Comment by localheinz** [comment]
Thank you, @svpernova09!

**Pull request merged**